Sming/Arch/Host/Components/esp_wifi/component.mk

```diff
@@ -1,3 +1,8 @@
+# Options to add for configuring host network behaviour
+CACHE_VARS				+= HOST_NETWORK_OPTIONS
+HOST_NETWORK_OPTIONS	?=
+SMING_TARGET_OPTIONS	+= $(HOST_NETWORK_OPTIONS)
```

Comment:
Maybe you can rename `SMING_TARGET_OPTIONS` to `CLI_TARGET_OPTIONS`. Let's use the prefix `SMING_` for important things like SMING_HOME and SMING_ARCH.